### DISCUSSION TITLE
v1.75.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3870] [Announcements]
# v1.75.0

## Breaking Changes ⚠️

> [!WARNING]
> - To disable machine learning now, use `IMMICH_MACHINE_LEARNING_ENABLED=false`
> (previously `IMMICH_MACHINE_LEARNING_URL=false`)
>
> - Sentence-Transformers is no longer used for CLIP models, users who set `MACHINE_LEARNING_CLIP_IMAGE_MODEL` or `MACHINE_LEARNING_CLIP_TEXT_MODEL` must migrate to one of the ViT-B models listed [here](https://clip-as-service.jina.ai/user-guides/benchmark/#size-and-efficiency), with the caveat that OpenCLIP models will require running CLIP on all images since the embeddings are incompatible. 

## Highlights

Welcome to release `v1.75.0` of Immich. There are many improvements in this version that you will like. Some of the key highlights include:

- Add the ability to use a configuration file for bootstrapping an Immich instance instead of using the administration web UI. [Documentation](https://immich.app/docs/install/config-file)
- Machine learning sub-jobs can now be configured via the administration web UI
- Slide show mode on the web

https://github.com/immich-app/immich/assets/27055614/bf2c7d0f-30a7-4808-94b0-2189db1862f1

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### Server
* fix(server): use camera white balance for raw images by @mertalev in https://github.com/immich-app/immich/pull/3806
* feat(server): asset entity audit by @fyfrey in https://github.com/immich-app/immich/pull/3824
* feat(web,server)!: configure machine learning via the UI by @jrasm91 in https://github.com/immich-app/immich/pull/3768
* feat(web, server): Ability to use config file instead of admin UI by @danieldietzler in https://github.com/immich-app/immich/pull/3836

### Web
* fix(web): ContextMenu unsatisfying UI behaviors by @flyotlin in https://github.com/immich-app/immich/pull/3787
* fix(web): album list padding by @waclaw66 in https://github.com/immich-app/immich/pull/3790
* fix(web): merge face thumbnail by @waclaw66 in https://github.com/immich-app/immich/pull/3822
* fix(web): shared link return 404 by @alextran1502 in https://github.com/immich-app/immich/pull/3791
* fix(web): cannot view publlic shared album by @alextran1502 in https://github.com/immich-app/immich/pull/3829
* refactor(web): shared link key auth by @jrasm91 in https://github.com/immich-app/immich/pull/3855
* fix(web): delete album consolidation by @waclaw66 in https://github.com/immich-app/immich/pull/3858
* feat(web): album and face menu dots visible on hover only by @waclaw66 in https://github.com/immich-app/immich/pull/3859
* feat(web): smaller album title by @waclaw66 in https://github.com/immich-app/immich/pull/3860
* feat(web): add link to external map in leaflet popup by @lefuturiste in https://github.com/immich-app/immich/pull/3847
* feat(web): slideshow mode by @alextran1502 in https://github.com/immich-app/immich/pull/3813
* chore(web): use proper text/drop shadow on tree-dots icon and face name by @daniele-athome in https://github.com/immich-app/immich/pull/3800
* chore(web): improve drop shadow on three-dots icon by @daniele-athome in https://github.com/immich-app/immich/pull/3835

### Machine learning
* chore(ml): make lazy loading default by @mertalev in https://github.com/immich-app/immich/pull/3797
* feat(ml)!: switch image classification and CLIP models to ONNX by @mertalev in https://github.com/immich-app/immich/pull/3809

### Mobile
* fix(mobile): missing conversion to UTC time zone by @james58899 in https://github.com/immich-app/immich/pull/3495
* chore(mobile): remove without thumbs by @jrasm91 in https://github.com/immich-app/immich/pull/3529

## New Contributors
* @flyotlin made their first contribution in https://github.com/immich-app/immich/pull/3787
* @james58899 made their first contribution in https://github.com/immich-app/immich/pull/3495
* @lefuturiste made their first contribution in https://github.com/immich-app/immich/pull/3847
* @danieldietzler made their first contribution in https://github.com/immich-app/immich/pull/3836

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.74.0...v1.75.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.75.0'>v1.75.0</a>.</em>

**Comment by 1nkoma** [comment]
My knowledge is very limited and started using Docker on a Raspberry only because of Immich. After this update my connection with my machine breaks after a few minutes. It happens after this update. I can see now 2 IP addresses. Do you think it can be related with the update?

*update - After stopping docker the linux runs ok, so indeed this last update is somehow breaking my connection

**Reply by Pheggas** [reply]
Please, discuss this on our Discord server to have better support coverage.
